cmd/mutate-image/README.md

```diff
@@ -0,0 +1,40 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+# Mutate Image Wrapper
+
+As part of the build, the output image needs to be mutated (annotated and labeled). This package contains Shipwright Build owned image mutate code, which is using the [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) in the background.
+
+## Features
+
+- Mutate the image with [annotations](https://github.com/opencontainers/image-spec/blob/main/annotations.md)
+- Mutate the image wth labels
+
+## Development
+
+### Run the CLI code
+
+- Run it locally:
+
+  ```sh
+  go run cmd/mutate-image/main.go \
+  --image $IMAGE \
+  --annotation "org.opencontainers.image.url=https://my-company.com/images" \
+  --label "maintaner=team@my-company.com"
+  ```
+   Multiple values for annotation and label can be provided as comma-separated values (`--label "description=image description,maintainer=team@my-company.com"`). If we are trying to mutate the image in a private registry, authentication to the registry should be done before running the command.
```

Comment:
Multiple values should be passed using multiple args otherwise a comma in the value would break it:

`--label "description=image description" --label "maintainer=team@my-company.com"`

I am not sure how `StringSliceVarP` behaves. I had expected it supports multiple args. Can you try?

Reply:
`StringSliceVarP` flags take comma-separated value as arguments and split them accordingly.
```// For example:
//   --ss="v1,v2" --ss="v3"
// will result in
//   []string{"v1", "v2", "v3"}
```

I have replaced the `StringSliceVarP` with `StringArray` to read the comma in the value and made changes in test cases accordingly.